tests/CacheCleanerIntegrationTest.php

```diff
@@ -0,0 +1,149 @@
+
+        ResponseCache::cacheCleaner()->setParameters(['foo' => 'bar'])->forget('/random');
+
+        $secondResponse = $this->get('/random?foo=bar');
+        $this->assertRegularResponse($secondResponse);
+
+        $this->assertDifferentResponse($firstResponse, $secondResponse);
+    }
+
+    /** @test */
+    public function it_can_forget_a_specific_cached_request_using_cache_cleaner_post()
+    {
+        config()->set('app.url', 'http://spatie.be');
+
+        $firstResponse = $this->post('/random');
+        $this->assertRegularResponse($firstResponse);
+
+        ResponseCache::cacheCleaner()->setMethod('POST')->forget('/random');
+
+        $secondResponse = $this->post('/random');
+        $this->assertRegularResponse($secondResponse);
+
+        $this->assertDifferentResponse($firstResponse, $secondResponse);
+    }
+
+    /** @test */
+    public function it_can_forget_several_specific_cached_requests_at_once_using_cache_cleaner()
+    {
+        $firstResponseFirstCall = $this->get('/random/1?foo=bar');
+        $this->assertRegularResponse($firstResponseFirstCall);
+
+        $secondResponseFirstCall = $this->get('/random/2?foo=bar');
+        $this->assertRegularResponse($secondResponseFirstCall);
+
+        ResponseCache::cacheCleaner()->setParameters(['foo' => 'bar'])
+            ->forget(['/random/1', '/random/2']);
+
+        $firstResponseSecondCall = $this->get('/random/1?foo=bar');
+        $this->assertRegularResponse($firstResponseSecondCall);
+
+        $secondResponseSecondCall = $this->get('/random/2?foo=bar');
+        $this->assertRegularResponse($secondResponseSecondCall);
+
+        $this->assertDifferentResponse($firstResponseFirstCall, $firstResponseSecondCall);
+        $this->assertDifferentResponse($secondResponseFirstCall, $secondResponseSecondCall);
+    }
+    /** @test */
+    public function it_can_forget_several_specific_cached_requests_at_once_using_cache_cleaner_post()
+    {
+        $firstResponseFirstCall = $this->post('/random/1');
+        $this->assertRegularResponse($firstResponseFirstCall);
+
+        $secondResponseFirstCall = $this->post('/random/2');
+        $this->assertRegularResponse($secondResponseFirstCall);
+
+        ResponseCache::cacheCleaner()->setMethod('POST')->forget(['/random/1', '/random/2']);
+
+        $firstResponseSecondCall = $this->post('/random/1');
+        $this->assertRegularResponse($firstResponseSecondCall);
+
+        $secondResponseSecondCall = $this->post('/random/2');
+        $this->assertRegularResponse($secondResponseSecondCall);
+
+        $this->assertDifferentResponse($firstResponseFirstCall, $firstResponseSecondCall);
+        $this->assertDifferentResponse($secondResponseFirstCall, $secondResponseSecondCall);
+    }
+
+
+    /** @test */
+    public function it_can_forget_a_specific_cached_request_using_cache_cleaner_suffix()
+    {
+        config()->set('app.url', 'http://spatie.be');
+
+        $user_id = 1;
```

Comment:
Please use `camelCase` for variable names.

Reply:
ok